apps/spruce/src/components/DayPicker/DayPicker.test.tsx

```diff
@@ -75,4 +75,16 @@ describe("daypicker", () => {
       false,
     ]);
   });
+
+  it("disables widget", async () => {
```

Comment:
nit. Can remove `async`!
```suggestion
  it("disables widget", () => {
```